frontend/src/pages/UserModal.tsx

```diff
@@ -0,0 +1,183 @@
+import {User} from "../models/User";
+import {BACKEND_API_URL} from "../constants";
+import {Link} from "react-router-dom";
+import {Avatar, Box, Dialog, DialogActions, DialogContent, DialogTitle, TextField, Typography} from "@mui/material";
+import {Colors} from "../assets/Colors";
+import {useState} from "react";
+import axios from "axios";
+import {HighlightOffRounded} from "@mui/icons-material";
+
+function UserListInterface(user: User) {
+    let icon = `${BACKEND_API_URL}/icon/by-id/${user.profile.icon_id}`;
+
+    const handleClick = () => {
+        setTimeout(() => window.location.reload(), 0);
```

Comment:
@MatheAndrei ai putea sa arunci o privire la chestia asta pls?

Se pare ca nu-si da re-render pagina (adica ramane pagina utilizatorului curent) atunci cand apesi pe chestia cu ``<Link to={{ pathname: `/profile/${user.id}`}} key={user.id} onClick={handleClick}>``, decat daca pui schema asta cu `handleClick()`. 

Dar pare kinda sussy schema 👀. M-as gandi ca ar trebui sa fie un mod sa faci fara schema.

Reply:
S-ar putea sa mearga cu modificarile Soniei